### PR TITLE
Added the new special: Kraken in the Ice.

### PR DESCRIPTION
**Pull request by Deepsloth**
Discussion: http://www.freeorion.org/forum/viewtopic.php?f=15&t=10173

Post v0.4.6.

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -6774,6 +6786,14 @@ Juggernaut Nest
 JUGGERNAUT_NEST_SPECIAL_DESC
 At some point young juggernauts will rise from this place.  With knowledge of [[tech SHP_DOMESTIC_MONSTER]], these mega-fauna could be domesticated.
 
+KRAKEN_IN_THE_ICE_SPECIAL
+Kraken in the Ice
+
+KRAKEN_IN_THE_ICE_SPECIAL_DESC
+'''With the knowledge of the technologies [[tech SHP_DOMESTIC_MONSTER]] and [[tech LRN_XENOARCH]], the [[predefinedshipdesign SM_WHITE_KRAKEN]] can be resurrected and controlled by building a [[buildingtype BLD_XENORESURRECTION_LAB]] on this planet.
```
Comment (dbenage-cx) [inline default/stringtables/en.txt:6732]: Not seeing any requirement or other mention of SHP_DOMESTIC_MONSTER.

**Review by dbenage-cx — CHANGES_REQUESTED** [review]
Description inconsistent

**Comment by dbenage-cx** [comment]
Curious if the extreme rarity is intended.  Very few games will have this actually placed on the map.

**Comment by Deepsloth** [comment]
I will add the check for SHP_DOMESTIC_MONSTER.

Have to test the rarity. Is it so extreme?

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -815,6 +815,12 @@ SM_KRAKEN_3_DESC
 
 Favored Environment: Gas Giants. Kraken nests are normally found in the orbit of Gas Giant planets, and Kraken can mature into larger forms in systems with Gas Giants.'''
 
+SM_WHITE_KRAKEN
+White Kraken
+
+SM_WHITE_KRAKEN_DESC
+'''The White Kraken is a white-colored prehistoric ancestor of the Kraken Space Monster.'''
```
Comment (Cjkjvfnby) [inline default/stringtables/en.txt]: No need quotes here

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: default/scripting/buildings/XENORESURRECTION_LAB.focs.txt [hunk 3]
```diff
@@ -35,6 +36,24 @@ BuildingType
...
+                Object id = Source.PlanetID
+                Planet
+                HasSpecial name = "KRAKEN_IN_THE_ICE_SPECIAL"
+            ]
+            activation = Source
+            effects = [
+                CreateShip designname = "SM_WHITE_KRAKEN" empire = Source.Owner
+                RemoveSpecial name = "KRAKEN_IN_THE_ICE_SPECIAL"
+                GenerateSitRepMessage
+                    message = "EFFECT_WHITE_KRAKEN_RESURRECTED"
```
Comment (dbenage-cx) [inline default/scripting/buildings/XENORESURRECTION_LAB.focs.txt:50]: `label` is now required, as of d55b16cf

**Review by dbenage-cx — CHANGES_REQUESTED** [review]

**Comment by dbenage-cx** [comment]
> Have to test the rarity. Is it so extreme?

My attempts with 10 different seeds resulted in 0 placements(200 systems, high specials).  I assume this is mostly from the rarity of a suitable tundra planet.
Increasing the limit results in some placement, but not 100% of the limit.  If I read it correctly, a spawnrate of 9.0 should give a 90% chance with low specials.

**Comment by adrianbroher** [comment]
Please don't merge master.

**Comment by Deepsloth** [comment]
How can i fix that? How can i avoid that in the future?

**Comment by adrianbroher** [comment]
> How can i fix that?

Use `git rebase` or `git pull --rebase` when you're on your topic branch.

> How can i avoid that in the future?

Don't use `git merge master` or `git pull` when you're on your topic branch.

=)

Seriously though: the problem with merging master into a topic branch caused us several times to re-introduce bugs, see also [the corresponding discussion](http://www.freeorion.org/forum/viewtopic.php?f=9&t=9595) about that topic and the introduction of the policy.

If you have trouble with the concept and the steps to approach this I suggest you to read the [Merging vs. Rebasing](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) tutorial from Atlassian.

**Comment by MatGB** [comment]
For this specific the easiest thing is for me to cherry pick it and push direct to master, I'll do it later.

**Comment by MatGB** [comment]
K, I've pushed the commits in directly and written up how I did it on the howto thread [Post](http://www.freeorion.org/forum/viewtopic.php?p=85827#p85827), as some of it is stuff I only learnt how to do very recently anyway. Cherry Picking is a very useful tool, especially when PR reviewing, I can put whatever I want into my Working branch and test it alongside my existing work easily.

**Comment by Vezzra** [comment]
Added the "status:merged" label to indicate that the contents of the PR actually have been merged (by cherry-picking).

**Pull request closed without merging**